### ISSUE TITLE
Add a "downloads" badge

### ISSUE THREAD

**Issue by bmitch** [bmitch/Codor#117]
Could go between the packagist and licence badges.

```
[![Packagist](https://img.shields.io/packagist/dt/bmitch/codor.svg)]()
```

**Comment by d35k** [comment]
pr #118

**Comment by campionfellin** [comment]
@d35k looks like you just beat me to it!

**Comment by bmitch** [comment]
@campionfellin I have another very similar issue in another package: https://github.com/bmitch/churn-php/issues/118 if you are interested.

**Comment by bmitch** [comment]
Fixed by #118

**Comment by campionfellin** [comment]
Yeah, I'll go ahead and do it there, thanks!
